science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
 ---
-page-title: "SB06: What does a red warning mean when the incidence is high?"
-page-description: "SB06: What does a red warning mean when the incidence is high?"
+page-title: "SB06: What does a red warning mean when incidence is high?"
+page-description: "SB06: What does a red warning mean when incidence is high?"
 page-name: science-blog-6
 page-name_de: science-blog-6
-author: CWA Team
-date: "2022-11-30"
+author: CWA-Team
+date: "2023-02-06"
+update: "last updated November 30, 2022 (German version)"
 layout: science
 ---
 
-An English version of this science blog article will follow in due course. Please take a look at the [German version](/de/science/2022-11-23-science-blog-6) until the English version is available.
+In this paper we want to examine the significance of a red warning versus a lower risk with or without a risk exposure. For this we consider the positivity rates of the test results, depending on the indicated risk. The data for this comes from the [data donation of the Corona-Warn-App](2021-10-15-science-blog-4/#3-privacy-preserving-analytics).
+
+In recent months, there has been criticism that in times of high incidence, a red warning is no longer meaningful. The risk of infection would exist regardless of whether there was a red warning in the CWA or not. Moreover, the frequency of red warnings is so high that the individual warning no longer has any significance. We want to show that this is not the case. In particular for rapid test results, which are often the first reason for a PCR test, the positivity rate with a red warning was significantly greater than with a green warning (low risk with risk exposure). Furthermore, we investigate whether these greater positivity rates with a red warning can be solely explained by local fluctuations in incidence.
 
 <!-- overview -->
 
-An English version of this science blog article will follow in due course. Please take a look at the [German version](/de/science/2022-11-23-science-blog-6) until the English version is available.
\ No newline at end of file
+1. [The most important findings in a nutshell: numbers, data and facts](#1-the-most-important-findings-in-a-nutshell-numbers-data-and-facts)
+2. [Positivity rates (nationwide)](#2-positivity-rates-nationwide)
+    1. [RAT](#21-rat)
+    2. [PCR](#22-pcr)
+3. [Local differences in incidence](#3-local-differences-in-incidence)
+4. [Positivity rates (local))](#4-positivity-rates-local)
+    1. [RAT](#41-rat)
+    2. [PCR](#42-pcr)
+5. [Positivity rate and incidence](#5-positivity-rate-and-incidence)
+    1. [RAT](#51-rat)
+    2. [PCR](#52-pcr)
+6. [Differences between urban and rural districts](#6-differences-between-urban-and-rural-districts)
+7. [Conclusion](#7-conclusion)
+8. [Continued further](#8-continued-further)
```

Comment:
German heading "So geht es weiter" is translated differently across science blog articles. "Continued further" would be yet another new variation. I would suggest to stick with one we already used in Science Blogs [2](https://www.coronawarn.app/en/science/2021-07-08-science-blog-2/), [3](https://www.coronawarn.app/en/science/2021-08-02-science-blog-3/) and [4](https://www.coronawarn.app/en/science/2021-10-15-science-blog-4/)

```suggestion
8. [What to expect in the future?](#8-what-to-expect-in-the-future)
```

Reply:
Added in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5